### ISSUE TITLE
Content Updates request for BDD and 526 form content page edits (Benefits and Memorials 1)

### ISSUE THREAD

**Issue by stephorkandatx** [department-of-veterans-affairs/va.gov-team#8895]
## Tasks

- [x] Draft content as part of IA/content strategy work (Danielle)
- [ ] Align on stakeholder alignment process (Danielle, Beth)
- [ ] Align with Peggy and product team (Danielle, Selina)
- [ ] Flow content changes into Drupal (Selina) - may move to next sprint depending on process of SME approval
- [ ] Finalize content per discussions/SME approvals & feedback (Selina, Beth) - may move to next sprint

### Background
*Provide a background of your tool/functionality.*
This is a request for the Benefits Delivery at Discharge functionality (allowing service members to file a claim online before they separate from the military) that is to be added into the 526 form functionality. Presently, the Benefits Delivery at Discharge application process is available only on ebenefits and via a paper form submission.

The current design for the proposed BDD form can be viewed here: https://vsateams.invisionapp.com/share/BUWOTNKFPK7

The findings from our usability testing research initiative can be found here:

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/disability-compensation-claim/bdd/BDD%20Research/Initial-usability-March2020/BDD-research-findings.md

### Request Type
*Select all applicable and provide details behind your request* 

           
- [ ] **Content support for a static public website page**
   * Is this to update existing static content? 
      * Yes

   * Provide a list of URL/s that need to be reviewed or updated. 
      * https://staging.va.gov/disability/how-to-file-claim/ (526 disability info/wizard page)
      * https://staging.va.gov/disability/how-to-file-claim/when-to-file/pre-discharge-claim/ (BDD info page)

** Further details **

#### 526 Disability Info Page

We believe this page can be improved in a couple key ways:

- There may be value in letting the user know exactly what types of claims they could file via the 526 form. There is excellent information outlining exactly what the Veteran or SM can file on the 526 via [this page](https://www.va.gov/disability/how-to-file-claim/when-to-file/). All disability filing types listed in the accordions on this page will be available via the 526 online form (we're getting original claims wrapped up now) and we will additionally be able to handle BDD via the 526 as well (est launch date in late June).
- There may be value in moving the 'action' button up a bit on the page - though we do understand that this is already in the works via the gating/wizard refactor your team is currently working on.

**if you would prefer us to supply a visual/design mock on what we're thinking for this, we can do that as well. Please let us know!**

#### BDD Info Page

- There may be value in condensing the information on this page - specifically removing the blue box and creating sections about certain eligibility (date ranges from separation date) and outlining what options are available for those specific dates.
- There may be value in nesting actions that the user can take for the date range they fall into (e.g. 'if you're 180-90 days out and meet the above requirements, you can file a BDD claim **online** or via the **paper form**." - links denoted in bold) 

**We plan on supplying a visual on this. We will upload to Invision and link to it next week (week of May 4)**

### Desired/target implementation date

We are anticipating a launch for BDD in late June. This request can be satisfied anytime before then.

**Comment by stephorkandatx** [comment]
@lunascoop ticket (housed in the new Epic) for Content work needed for this request. I will circle back to reformat the tasks/AC as best I can. I will want to meet with you once you sync with the team and we can complete the ask. If there are additional needs (other tickets) let's connect so I can get that documented. Thank you!

**Comment by stephorkandatx** [comment]
Content strategy, Drupal work,

**Comment by lunascoop** [comment]
Met w/ Mikki and Danielle yesterday. IA needs to finalize strategy before moving forward w/content work.

**Comment by MickinSahni** [comment]
Pulling out of sprint 25 until unblocked

**Comment by DanielleThierryUSDSVA** [comment]
@lunascoop @MickinSahni Quick update on this. IA/content strategy work tracking in this issue: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/10896

Should be finalized next week to unblock this issue. Thanks!

**Comment by DanielleThierryUSDSVA** [comment]
@bethpottsVADEPO @lunascoop IA/content strategy is complete for this now so we can build out the full content. 

[Consolidating Disability how to file and BDD sections](https://app.zenhub.com/files/133843125/c8cd6aa6-c650-44da-948e-c092c3dd3a1a/download)

@bethpottsVADEPO, we should talk about how we want to present this to SMEs...whether we want to use Word doc (or even a quick mockup) to get alignment before building out in Drupal. And we'll need to determine with Sandy which SMEs need to be involved in review.

**Comment by DanielleThierryUSDSVA** [comment]
@bethpottsVADEPO Can we connect on this one today to go over Q above? Thanks!

**Comment by MickinSahni** [comment]
@DanielleThierryUSDSVA  to break this epic out into tickets

**Comment by DanielleThierryUSDSVA** [comment]
FYSA @MickinSahni I haven't had a chance to do this breakout yet. Will do for next sprint. Thanks!

**Comment by mnorthuis** [comment]
@DanielleThierryUSDSVA  @bethpottsVADEPO  Wanted to check in on the content updates related to the BDD launch.  The BAM1 team is planning to launch BDD next week.

**Comment by mnorthuis** [comment]
Updated the IA info based on follow up conversations around what work is needed to best support the BDD launch and what can be postponed.

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/ia-reviews/bam-benefits-at-discharge.md

Redirect request to support above IA work has been submitted: #12825

**Comment by brianalloyd** [comment]
@DanielleThierryUSDSVA I'm still tracking this on the PW in-take and just wanted validate it's still an open and active requirement?  @brianalloyd

**Comment by DanielleThierryUSDSVA** [comment]
Hi @brianalloyd. Thanks for checking! We can go ahead and close this for now. We have some updates we'd like to make, but they're not tied to the launch. Thanks again.

**Comment by brianalloyd** [comment]
Closing per crosscheck with @DanielleThierryUSDSVA.
